### ISSUE TITLE
Enhancement: Allow user to specify the the sub menu selector

### ISSUE THREAD

**Issue by coxrichuk** [joeldbirch/superclick#7]
Would it be worth adding an option to allow the user to specify the sub menu?

This would be helpful when laying out multiple columns within a sub menu?

I already have the code for this if you would like to take a look on a pull request?

**Comment by joeldbirch** [comment]
Sounds like the `popUpSelector` feature from Superfish. Looks like I haven't folded that back into Superclick yet.

**Comment by coxrichuk** [comment]
That would make sense as I can't see it being used anywhere.

**Comment by joeldbirch** [comment]
I just released Superclick v1.1.0 which adds this feature. There is a MegaMenu example html file for you to see it in action using the default selector `'ul,.sf-mega'` (which is automatically scoped to children of Superclick `<li>` elements) but you can alter that when you initialise Superclick, like this:

``` js

$('#my-menu').superclick({
  popUpSelector: '.some-other-element'
});

```
